RELEASE.md

```diff
@@ -1,5 +1,7 @@
 # Release process
 
+See [Platform Release doc](https://github.com/infinyon/fluvio-cloud-ops/blob/master/docs/release/RELEASE.md) for integrated platform release process steps.
+
```

Comment:
maybe we should not mention this here?

Reply:
I'm was undecided about this, but it put it in for discussion.  We should only have one release procedure. Maybe we should remove most of this RELEASE file and depend on the fc-ops one? It was nicer to only have to look at one prodcedure (+ issues).

Reply:
remove cloud ops release

Reply:
removed!